x-pack/elastic-agent/pkg/fleetapi/acker/fleet/fleet_acker.go

```diff
@@ -114,6 +114,7 @@ func constructEvent(action fleetapi.Action, agentID string) fleetapi.AckEvent {
 	}
 
 	if a, ok := action.(*fleetapi.ActionApp); ok {
+		ackev.InputType = a.InputType
```

Comment:
Is there any unit tests for this path? Would be great to add a test for this.

Reply:
Rewrote acker unit tests, to simplify them, add coverage and eliminate the go routines leak.
This function is fully covered now:
<img width="819" alt="Screen Shot 2022-02-24 at 11 53 39 AM" src="https://user-images.githubusercontent.com/872351/155571174-21e7935d-c00d-44f0-b612-c81f670cd5e0.png">